docs/resources/marker_setting.md

```diff
@@ -0,0 +1,38 @@
+# Resource: honeycombio_marker_setting
+
+Creates a marker setting. For more information about marker settings, check out [Marker Settings API](https://docs.honeycomb.io/api/marker-settings/).
+
+## Example Usage
+
+```hcl
+variable "dataset" {
+  type = string
+}
+
+variable "color" {
+  type = string
+}
+
+resource "honeycombio_marker_setting" "markerSetting" {
+  type = "deploy"
+  color = "${var.color}"
```

Comment:
Given that `color` requires a hex color code I'd like to see that in this example instead of the variable to help make it more clear. So, maybe remove lines 12-14 and set this to `color = "#DF4661"`